### PR TITLE
Remove quantum plots

### PR DESCRIPTION
**Pull request by johannesjmeyer**
This PR replaces the quantum-plots notebook with the better maintained and easier-to use library rsmf, as discussed in #121.

Changes:
- Removed quantum-plots.ipynb and quantum-plots.mplstyle
- Removed mentions of quantum-plots in the template
- Added example-plot.py to showcase how rsmf is used to create Figure 1 in the template
- Added mention to rsmf and the example in the template

After David is done with the bibstyle, we can bump the version and publish to CTAN.

### CODE REVIEW

**Comment by johannesjmeyer** [comment]
Ping @cgogolin

File: quantum-template.tex [hunk 1]
```diff
@@ -181,8 +181,8 @@ \subsection{References and footnotes}
...
-We strongly encourage authors to use the vector based PDF format for plots.
+We strongly encourage authors to use the vector based PDF format for plots. The Python package \href{https://pypi.org/project/rsmf/}{rsmf} (``right-size my figures'') together with the widely used plotting library \href{https://matplotlib.org/}{matplotlib} can be used to create plots that integrate seamlessly with the \texttt{quantumarticle} document class. In the Quantum \href{https://github.com/quantum-journal/quantum-journal/}{GitHub repository}, you find the script \texttt{example-plot.py} that shows how these tools were used to create Figure~\ref{fig:figure1}.
```
Comment (cgogolin) [inline quantum-template.tex]: My only request: Can we make "example-plot.py" a direct download link to the file from the master branch? Probably best via https://raw.githubusercontent.com/quantum-journal/quantum-journal/master/...

**Review by cgogolin — CHANGES_REQUESTED** [review]

File: quantum-template.tex (hunk 1 above)
Comment (johannesjmeyer) [reply quantum-template.tex]: Done :)

**Comment by johannesjmeyer** [comment]
@cgogolin you can merge if you're happy

**Review by cgogolin — APPROVED** [review]

**Pull request merged**